presto-spi/src/main/java/com/facebook/presto/spi/statistics/PlanStatistics.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.spi.statistics;
+
+import static java.util.Objects.requireNonNull;
+
+public class PlanStatistics
+{
+    private static final PlanStatistics EMPTY = new PlanStatistics(Estimate.unknown(), Estimate.unknown(), 0);
+
+    private final Estimate rowCount;
```

Comment:
Can be expanded later, but we might want the possibility to provide other kinds of stats too

Reply:
Yeah we'll add more here later